frontend/src/employee-frontend/components/person-profile/PersonVoucherValueDecisions.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { orderBy } from 'lodash'
+import { faChild } from 'lib-icons'
+import { Loading, Result } from 'lib-common/api'
+import { formatDate } from 'lib-common/date'
+import Loader from 'lib-components/atoms/Loader'
+import CollapsibleSection from 'lib-components/molecules/CollapsibleSection'
+import { Table, Tbody, Td, Th, Thead, Tr } from 'lib-components/layout/Table'
+import { useTranslation } from 'employee-frontend/state/i18n'
+import { VoucherValueDecisionSummary } from 'employee-frontend/types/invoicing'
+import { getPersonVoucherValueDecisions } from 'employee-frontend/api/invoicing'
+import { formatCents } from 'employee-frontend/utils/money'
+
+interface Props {
+  id: string
+  open: boolean
+}
+
+export default React.memo(function PersonVoucherValueDecisions({
+  id,
+  open
+}: Props) {
+  const { i18n } = useTranslation()
+
+  const [voucherValueDecisions, setVoucherValueDecisions] = useState<
+    Result<VoucherValueDecisionSummary[]>
+  >(Loading.of())
+
+  const loadDecisions = useCallback(() => {
+    setVoucherValueDecisions(Loading.of())
+    void getPersonVoucherValueDecisions(id).then((response) => {
+      setVoucherValueDecisions(response)
+    })
+  }, [id, setVoucherValueDecisions])
```

Comment:
would `useRestApi` be better?

Reply:
It should make any functional difference.